File: ExampleMvvm/librarycommon/src/main/java/com/degde/library/library/login/model/ModelLoginRequest.java
package com.degde.library.library.login.model;

public class ModelLoginRequest {

    private String identity_provider;
    private String username;
    private String password;

    public String getIdentity_provider() {
        return identity_provider;
    }

    public void setIdentity_provider(String identity_provider) {
        this.identity_provider = identity_provider;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}

File: ExampleMvvm/corekotlin/src/main/java/com/core/mvvm/login/model/ModelLoginRequest.kt
package com.core.mvvm.login.model

class ModelLoginRequest {
    private var identity_provider: String? = null
    private var username: String? = null
    private var password: String? = null

    fun getIdentity_provider(): String? {
        return identity_provider
    }

    fun setIdentity_provider(identity_provider: String?) {
        this.identity_provider = identity_provider
    }

    fun getUsername(): String? {
        return username
    }

    fun setUsername(username: String?) {
        this.username = username
    }

    fun getPassword(): String? {
        return password
    }

    fun setPassword(password: String?) {
        this.password = password
    }
}